<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anemometer Positioning System (APS)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>University of Minnesota.png</normaloff>University of Minnesota.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="APSsetupGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>APS Setup</string>
    </property>
    <widget class="QComboBox" name="comPortChoice">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>106</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="QLabel" name="comPortLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Microcontroller COM Port:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="findPlaneLimits">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>110</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Find Plane Limits</string>
     </property>
    </widget>
    <widget class="QLabel" name="APSsetupStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Please Set Up the APS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dataSavingSettingsGroup">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>531</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Data Saving Settings</string>
    </property>
    <widget class="QLineEdit" name="dataPathBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>109</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dataPathLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Data File Name and Location:</string>
     </property>
    </widget>
    <widget class="QLabel" name="dataPathStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Please Select a Valid File</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="dataPathBrowse">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="operationModeGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>451</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Operation Mode</string>
    </property>
    <widget class="QRadioButton" name="manualModeRadio">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Manual Mode</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="autoModeRadio">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Automatic Mode</string>
     </property>
    </widget>
    <widget class="QLabel" name="operationModeStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Operation Not Set Up</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="xOperationLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="yOperationLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="xOperationChoice">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="yOperationChoice">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="operationStart">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="timeOperationChoice">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="timeOperationLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="maximumXlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Maximum X: Not Defined</string>
    </property>
   </widget>
   <widget class="QLabel" name="maximumYlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Maximum Y: Not Defined</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentXlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current X: Unknown</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentYlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current Y: Unknown</string>
    </property>
   </widget>
   <widget class="QLabel" name="APSlabeledPicture">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>611</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;APS_labeled_resized.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSlider" name="ySlider">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>31</width>
      <height>531</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="xSlider">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>720</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="emergencyStopButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>710</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>STOP APS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1145</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="userManualOption"/>
    <addaction name="aboutOption"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="userManualOption">
   <property name="text">
    <string>User Manual</string>
   </property>
  </action>
  <action name="aboutOption">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="APS.qrc"/>
 </resources>
 <connections/>
</ui>
